server/aesdsocket.c

```diff
@@ -1,331 +1,580 @@
-/*
- * Author: Pranav Bharadwaj
- * Date: 09/30/2022
- * Formatted for space using codebeautify.org
- * References:
- 
- 1) Getting IPv4 address: https://stackoverflow.com/questions/1276294/getting-ipv4-address-from-a-sockaddr-structure
- 2) https://beej.us/guide/bgnet/html/#audience
- 3) https://www.tutorialspoint.com/c_standard_library/c_function_strerror.htm 
-*/
-
+/*Assignment 6 aesdsocket implementation*/
```

Comment:
@prvbharadwaj1731 you should remove the `server/aesdsocket` binary from your repo.